src/utils/hooks/useAuthErrorListener.ts

```diff
@@ -5,6 +5,7 @@ import { useDispatch } from 'react-redux';
 import { setNotification } from '../../redux/notificationSlice';
 
 enum AuthErrorCode {
+  Cancelled = 'User cancelled OAuth flow',
```

Comment:
Føles veldig skittent å sjekke mot en tekstverdi her, men ser ingen bedre løsning. Det vil brekke om/når Amplify endrer teksten for dette da..

<img width="352" height="115" alt="Image" src="https://github.com/user-attachments/assets/fd3e6212-9c94-4a8c-8f1f-5c8419d9a166" />